src/components/app.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import LocationDetails from './location-details';
+import ForecastSummaries from './forecast-summaries';
+import ForecastDetails from './forecast-details';
+import axios from 'axios';
+import SearchForm from './search-form';
+import '../styles/app.scss';
+
+const url = 'https://mcr-codes-weather.herokuapp.com/forecast';
+
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedDate: 0,
+      forecasts: [],
+      location: {
+        city: '',
+        country: '',
+      },
+      errorMessage: '',
+      searchText: '',
+    };
+  }
+
+
```

Comment:
It may be worth taking the initial city out of the `componentDidMount()` and storing it within state. You could even pass it in as a prop from `index.js`. This way you would have an easier time of expanding the app to detect the location of the user.